tests/main/auto-refresh-gating-from-snap/task.yaml

```diff
@@ -43,6 +46,12 @@ execute: |
   MATCH "pending: none" < pending.log
   NOMATCH "version:" < pending.log
 
+  echo "Check that --proceed fails if nothing is held by the snap initially"
+  "$SNAP_NAME".proceed 2>&1 | MATCH "no snaps are held by snap \"$SNAP_NAME\""
+
+  echo "And it works with the snap that doesn't have gate-auto-refresh hook"
+  "$NOHOOK_SNAP".proceed 2>&1 | MATCH "no snaps are held by snap \"$NOHOOK_SNAP\""
```

Comment:
I'm probably reading it wrong, but I see that both of the above commands return the same error message, yet the comments claim that the first one is failing, and the second one is succeeded. :-)

Reply:
Updated.

Reply:
Thanks!